src/utils/gameBoardHelper.ts

```diff
@@ -0,0 +1,157 @@
+import { Letter, PlayerIndex, SparseArray } from "../types";
+
+export function verticalSlice<T>(
+  array: SparseArray<T>,
```

Comment:
Because it works with any `SparseArray` that's holding data for 2D square array in 1D array given the 2D array's size. If anything should be renamed here, it should be `boardSize` -> `arraySize`